packages/page-staking/src/Performance/Performance.tsx

```diff
@@ -17,17 +16,12 @@ import BlockProductionCommitteeList from './BlockProductionCommitteeList.js';
 import Summary from './Summary.js';
 import { parseSessionBlockCount } from './useCommitteePerformance.js';
 
-interface Props {
-  session: number,
-  era: number,
-}
-
 export interface EraValidatorPerformance {
   validatorPerformance: ValidatorPerformance;
   isCommittee: boolean;
 }
 
-function Performance ({ era }: Props): React.ReactElement<Props> {
+function Performance (): React.ReactElement<null> {
```

Comment:
Return type can be omitted and will be inferred from function body
```suggestion
function Performance (): {
```

Reply:
good point, changed